src/lib/x509/asn1_alt_name.cpp

```diff
@@ -56,7 +56,8 @@ void AlternativeName::add_othername(const OID& oid, std::string_view value, ASN1
    if(value.empty()) {
       return;
    }
-   multimap_insert(m_othernames, oid, ASN1_String(value, type));
+   ASN1_String str(value, type);
+   m_othernames.insert(std::make_pair(oid, str));
```

Comment:
If you use `emplace`, you could even save the `std::make_pair` and end up with more or less the same line:

```suggestion
   m_othernames.emplace(oid, ASN1_String{value, type}));
```